data/strings/types_strings.txt

```diff
@@ -5256,6 +5338,46 @@
     zh-Hans = 国家公园
     zh-Hant = 國家公園
 
+  [type.boundary.protected_area]
+    en = Protected Area
+    ar = منطقة محمية
+    be = Запаведная зона
+    bg = Защитена местност
+    ca = Àrea Protegida
+    cs = Chráněné území
+    da = Beskyttet område
+    de = Geschützter Bereich
+    el = Προστατευόμενη Περιοχή
+    es = Área protegida
+    et = Kaitseala
+    eu = Babestutako Eremua
+    fa = منطقه حفاظت شده
+    fi = Suojeltu alue
+    fr = Zone protégée
+    he = אזור מוגן
+    hu = Védett terület
+    id = Kawasan Lindung
+    it = Area protetta
+    ja = 保護区域
+    ko = 보호구역
+    mr = संरक्षित क्षेत्र
+    nb = Vernet område
+    nl = Beschermd gebied
+    pl = Strefa chroniona
+    pt = Área protegida
+    pt-BR = Área protegida
```

Comment:
Одинаковые строки в субязыках удаляет наш скрипт clean strings

Reply:
Я запустил ./tools/python/clean_strings_txt.py -s  и ничего не изменилось.
Вроде как мы должны указывать и "под-языки" иначе что-то глючило, не?

Reply:
Я патчил twine, он автоматом должен брать в айосе субязык из базового. А на андроиде и без патча система бкрёт и работает.

Reply:
Not sure what happened, but on Android the PT-BR strings were not generated from pt.

Reply:
I removed duplicates (where pt equals pt-BR) because Android system makes this fallback itself.
If the system language is pt-BR, it takes strings from pt (if no appropriate value in pt-BR). @biodranik right?

Reply:
Ah ok! I remember that before the python script would do that from strings.txt, for both Android and iOS. Now I only see those strings in iOS files, hence the warning. Thanks for the explanation!

Reply:
Right.